src/CLI/clitree/cli-xml/interface.xml

```diff
@@ -143,7 +143,7 @@ limitations under the License.
         elif test "${if-subcommands}" = "Vlan"; then&#xA;
             if test "${vlan-id}" = ""; then&#xA;
                 #Note: TODO, Discussion needed
```

Comment:
Newline character `&#xA;` is required at the end of line 145 also. Without that CLI preprocessor merges 145 and 146 into one line - and it becomes a bash comment.